src/chains/ethereum/console.log/src/maybe-get-logs.ts

```diff
@@ -0,0 +1,72 @@
+import { BN } from "ethereumjs-util";
+import { WORD_SIZE } from "./handlers";
+import { signatureMap } from "./signatures";
+
+const CONSOLE_ADDRESS = new BN(
+  Buffer.from([
+    0x63, 0x6f, 0x6e, 0x73, 0x6f, 0x6c, 0x65, 0x2e, 0x6c, 0x6f, 0x67
+  ])
+);
+
+/**
+ * The purpose of this type is to help maintainers visualize the layout of
+ * the `stack`.
+ */
+type LogsStack = [
+  ...rest: BN[],
+  inLength: BN,
+  inOffset: BN,
```

Comment:
```suggestion
  memoryLength: BN,
  memoryOffset: BN,
```
Just for consistency with the name changing elsewhere.

Reply:
:+1: It might be worth understanding that the names here came from the EthereumJS VM, which generates the `event` this type is applied to.